views/forms/form_success.php

```diff
@@ -0,0 +1,51 @@
++<?php
++/**
++ * PEP Capping 2017 Algozzine's Class
++ *
++ * Success page for form submissions
++ *
++ * The success page is displayed when a user has properly submitted a form.
++ * They will be redirected here once the information is successfully submitted into the database.
++ * In its current state, it is used for showing functionality, and is not actually called on success of database submission,
++ * just success in data validation
++ *
++ * @author Christian Menk
++ * @copyright 2017 Marist College
++ * @version 1.0.0
++ * @since 1.0.0
++ */
+include('header.php');
+
+global $params, $route, $view;
+
+?>
+
+    <div style="width: 100%">
+        <?php
+        if(isset($_GET['type'])) {
+            $form_type = $_GET['type'];
```

Comment:
See the comment I made on the `self_referral_form.php` file about using `$_SESSION['form-type']`. Then instead of the `$_GET['type']` you have here do something like the following:

```php
if (isset($_SESSION['form-type'])) {
    $form_type = $_SESSION['form-type'];
    unset($_SESSION['form-type']);
    // ...
}